content.org

```diff
@@ -611,7 +649,7 @@ List of basic types:
 - first returns the first item in the collection.
 #+BEGIN_SRC clojure
 (first [5 10 15])
-(nth [5 10 15] 0)
+;; TODO write using nth function?
```

Comment:
I think we can keep this around. Seems like it's just to reinforce how nth works. Applying something we learned right away in a different context.

Reply:
Yes, but we will ask this to the audience. Thats why I have removed the solution.

Reply:
Yes, but we will ask this to the audience. Thats why I have removed the solution.